util/app.h

```diff
@@ -37,8 +37,8 @@
 // copybara:import_next_line:gemma_cpp
 #include "gemma.h"
 // copybara:import_next_line:gemma_cpp
```

Comment:
The next_line comment must also be removed whenever we remove/reorder an include of the files in gemma_cpp.
FYI this will no longer be necessary when source files have been moved to gemma/, as is planned.